docs/document/content/user-manual/shardingsphere-proxy/startup/graalvm-native-image.cn.md

```diff
@@ -0,0 +1,124 @@
++++
+title = "构建 GraalVM Native Image(Alpha)"
+weight = 2
++++
+
+## 背景信息
+
+本节主要介绍如何通过 `GraalVM` 的 `native-image` 组件构建 ShardingSphere-Proxy 的 `Native Image` 和对应的 `Docker Image`
+。
+
+## 注意事项
+
+- ShardingSphere Proxy 尚未准备好与 GraalVM Native Image 集成。
+  其在 https://github.com/apache/shardingsphere/actions/ 存在每日构建的任务用于测试构建。
+
+- 若你发现构建过程存在缺失的 GraalVM Reachability Metadata,
+  应当在 https://github.com/oracle/graalvm-reachability-metadata 打开新的 issue ，
+  并提交包含 ShardingSphere 自身或依赖的第三方库缺失的 GraalVM Reachability Metadata 的 PR。
+
+- ShardingSphere 的 master 分支尚未准备好处理 Native Image 中的单元测试,
+  需要等待 Junit 5 Platform 的集成，你总是需要在构建 GraalVM Native Image 的过程中，
+  加上特定于 `GraalVM Native Build Tools` 的 `-DskipNativeTests` 或 `-DskipTests` 参数跳过 Native Image 中的单元测试。
```

Comment:
I will introduce Junit 5 in another issue, the related changes after combining with GraalVM Native Build Tools are too weird.